arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -252,7 +293,48 @@
           "200": {

```

Comment:
Please add descriptions
get: "Lists all the container registries available under the subscription."